src/symbols.c

```diff
@@ -1314,20 +1318,20 @@ static void update_tree_tags(GeanyDocument *doc, GList **tags)
 
 	/* Build hash tables holding tags and parents */
 	/* parent table is GHashTable<tag_name, GTree<line_num, GtkTreeIter>> */
```

Comment:
The comment should be updated to make it clear it's tag with context when available.